Sources/DistributedActors/ClusterSystem.swift

```diff
@@ -330,7 +330,7 @@ public class ClusterSystem: DistributedActorSystem, @unchecked Sendable {
                 systemStream: true,
                 customBehavior: ClusterEventStream.Shell.behavior
             )
-            let clusterRef = try! cluster.start(system: self, clusterEvents: clusterEvents) // only spawns when cluster is initialized
+            let clusterRef = try! await cluster.start(system: self, clusterEvents: clusterEvents) // only spawns when cluster is initialized
```

Comment:
The changes in this PR have disrupted the shutdown flow, because the error has come back:

```
ERROR: Cannot schedule tasks on an EventLoop that has already shut down. This will be upgraded to a forced crash in future SwiftNIO versions.
```

Reply:
Okey :-/

I think for this PR feel free to ignore those and I'll see about it separately again 👍 
We need some more solid way to prevent this...

Reply:
This helps: https://github.com/apple/swift-distributed-actors/pull/977/commits/810cf91841c9b0dc38e6986a2a2336f959118816